### ISSUE TITLE
ksystemlog is missing/unpackaged in plasma5

### ISSUE THREAD

**Issue by cx405** [NixOS/nixpkgs#37666]
## Issue description
ksystemlog is Qt-based systemd log viewer.
```
# nix-locate ksystemlog
maia-icon-theme.out                             109,053 r /nix/store/gx8l1gzf1abaycr3r942psbjxbassdmf-maia-icon-theme-2016-09-16/share/icons/maia/apps/scalable/ksystemlog.svg
libsForQt5.knewstuff.bin                          5,321 r /nix/store/flnfg5dpl5p0qil5xnsn8y1hi2l38v8d-knewstuff-5.43.0-bin/share/kf5/kmoretools/presets-kmoretools/org.kde.ksystemlog.desktop
```

However, pulling "knewstuff" does not add it into the system.

### Steps to reproduce
```
$ nix-env -i knewstuff
$ whereis ksystemlog
ksystemlog:

```

## Technical details

- system: `"x86_64-linux"`
 - host os: `Linux 4.14.27, NixOS, 18.03pre131327.0e7c9b32817 (Impala)`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.0`
 - channels(root): `"nixos-18.03pre131327.0e7c9b32817, nixos17.09-17.09.3194.edf1079d43f"`
 - nixpkgs: `/nix/var/nix/profiles/per-user/root/channels/nixos/nixpkgs`

**Comment by peterhoeg** [comment]
Fixed in c1714f57880.

**Comment by cx405** [comment]
@peterhoeg  **Thank you very much!!**
